doc/sphinx-guides/source/developers/windows.rst

```diff
@@ -16,3 +16,91 @@ See the `post <https://groups.google.com/g/dataverse-dev/c/utqkZ7gYsf4/m/4IDtsvK
 - You must have jq installed: https://jqlang.github.io/jq/download/
 
 One the above is all set you can move on to :doc:`/container/dev-usage` in the Container Guide.
+
+Running Dataverse in Windows WSL
+--------------------------------
+
+It is possible to run Dataverse in Windows 10 and 11 through WSL (Windows subsystem for Linux)
+
+Install WSL
+~~~~~~~~~~~
+If you have Docker already installed, you should already have WSL installed, otherwise open PowerShell and run:
+
+.. code-block:: powershell
+  
+   wsl --install
+
+If you already had WSL installed you can install specific linux distribution:
+
+See the list of possible distributions:
+
+.. code-block:: powershell
+
+  wsl --list --online
+
+Choose the distribution you would like. Then run the following command. Notice that this installation of dataverse was tried with ubuntu distribution.
+
+.. code-block:: powershell
+
+  wsl --install -d <Distribution Name>
+
+You will be asked to create a linux user.
+After installation of Linux check that you have internet connection:
+
+.. code-block:: bash
+
+  ping www.google.com
+
+If you do not have internet connection try add in ``/etc/wsl.conf``
+
+.. code-block:: bash
+  
+  [network]
+  generateResolvConf = false
+
+Also in /etc/resolv.conf add
+
+.. code-block:: bash
+
+  nameserver 1.1.1.1
+
+Now you can install all the tools one usually uses in Linux. For example, it is good idea to run update:
+
+.. code-block:: bash
+
+   sudo apt update
+   sudo apt full-upgrade -y
+
+Install Dataverse
+~~~~~~~~~~~~~~~~~
+
+Now you can install Dataverse in WSL following the instructions for :doc:`classic-dev-env`
```

Comment:
Classic? Interesting. You aren't using Docker?

Reply:
Yes, as you know I tried to run Dataverse using Docker, but it did not work for develop branch, but it did seem to work for demo. So I decided to try WSL. But to run Docker inside WSL seemed to be too much, one virtualization on top of another. Also classic installation gives more control such as direct access to database and solr. Surprisingly, WSL works very well for dataverse, I do not see much difference in performance between my ubuntu machine and WSL ubuntu on Windows. I do have better UI windows experience though.